config/default/kustomization.yaml

```diff
@@ -15,6 +15,7 @@ namePrefix: kubestellar-
 #    someName: someValue
 
 resources:
+- ../crd
```

Comment:
To avoid more confusion in the future I think we should remove that line 18 (the crds are not incorporated in the crd-level kustomization file, but it should be easier to understand if we disable from here)

Reply:
ok, done

Reply:
but not pushed

Reply:
@MikeSpreitzer it was already excluded, this PR was putting it back, but since @francostellari took it out now it does not show in the changed files - see https://github.com/kubestellar/kubestellar/blob/a7d1e58feebd3d661811c3a1dc93f46fa923e5fc/config/default/kustomization.yaml

Reply:
Thanks for the explanation.
/unhold